deploy-as-code/helm/charts/ifix-services/ifix-migration-toolkit/templates/deployment.yaml

```diff
@@ -0,0 +1,2 @@
+# deployment.yaml
+{{- template "common.deployment" . -}}
```

Comment:
Is this a one time job or a full time server?

Reply:
Full time server

Reply:
Then this would also need a service and optionally an ingress.